hack/update-all.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+${TESTINFRA_ROOT}/hack/update-bazel.sh
+${TESTINFRA_ROOT}/hack/update-gofmt.sh
```

Comment:
Is `update-gofmt.sh` mutating the actual files or is it sending its output to stdout? If the latter, I am not sure how useful it is. Maybe a follow-up issue to fix it to actually update.

Reply:
It's mutating them if they need formatting (it has the -w flag).